channels/italiafilm.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------
+# Canale per 'italifilm'
+# ------------------------------------------------------------
+
+from core import support, httptools
+from core.support import info
+from platformcode import logger, config
+from core import scrapertools
+
+host = config.get_channel_url() 
+headers = [['Referer', host]] 
+
+
+@support.menu
+def mainlist(item):
+    menu = [('Per Genere', ['', 'list', 'genere']),
+               ('Al Cinema', ['/cinema/', 'list', 'film']),
+               ('Top del Mese', ['/top-del-mese.html', 'list', 'film']),
+               ('Sottotitolati', ['/sub-ita/', 'list', 'film'])
+           ]
+    search = ''
+
+    return locals() 
+
+
+@support.scrape
+def list(item):
+
+    if item.args == 'genere':
+        patronBlock = r'<ul class="sub-menu">(?P<block>.*?)</ul>'
+        patronMenu = r'<li><a href="(?P<url>[^"]+)">(?P<title>[^<]+)'
+        action = 'peliculas'
+    elif item.args == 'film':
+        patronBlock = r'<div class="entry-summary">(?P<block>.*?)</div>'
```

Comment:
Funziona, ma può essere evitato (così esegue la regex della variabile patron per ogni risultato di patronBlock). Includila dentro patron.

patronBlock è inteso per essere usato quando la parte che si vuole ottenere è divisa in sezioni (es: stagione 1 con i suoi episodi, stagione 2 ecc..)